lib/mquery.js

```diff
@@ -1257,13 +1258,13 @@ Query.prototype.sort = function(arg) {
   }
 
   // .sort('field -test')
-  if (1 === arguments.length && 'string' == type) {
-    arg = arg.split(/\s+/);
+  if (1 === arguments.length && 'string' === type) {
+    arg = arg.split(whitespaceRE);
```

Comment:
Worth it to check whether there is whitespace in the string before splitting? split() tends to be super slow